app/components/welcome.jsx

```diff
@@ -0,0 +1,25 @@
+import '../scss/hello-react';
+import React from 'react';
+import { Router, Route, Link } from 'react-router'
+
+export default class extends React.Component {
+    render() {
+        return (
+            <div>
+                <header>
+                    <h1 className="hello-react">Welcome to Team Hydra</h1>
+                </header>
+                <aside>
+                    <ul>
+                        <li><Link to="/">Welcome</Link></li>
```

Comment:
i get this error when i try it - Warning: the query argument to createHref is deprecated; use a location descriptor instead , not good :)